multipaz-compose/src/commonMain/kotlin/org/multipaz/compose/camera/Camera.kt

```diff
@@ -2,18 +2,24 @@ package org.multipaz.compose.camera
 
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.ImageBitmap
 
 /**
  * Custom composable taking care of the camera operations initialization and camera preview composition.
- * Should be considered a placeholder for the future Camera component used throughout the application for a variety
- * of the hardware camera tasks like QR code scanning and face recognition.
+ * Should be considered the starting point for the future Camera components used throughout the application
+ * for a variety of the live image input tasks like QR code scanning and face recognition.
  *
- * @param cameraSelector The initial camera selection to use.
- * @param showPreview Whether to show the camera preview or process the data silently.
- * @param modifier The external modifier to apply to the composable.
+ * @param modifier The composition modifier to apply to the composable.
+ * @param cameraSelection The desired hardware camera ID to use.
+ * @param showCameraPreview Whether to show the native camera preview (Camera SDK bound).
+ * @param captureResolution The desired resolution to use for the camera capture.
+ * @param onFrameCaptured A callback to invoke when a frame is captured.
  */
 @Composable
 expect fun Camera(
-    cameraSelector: CameraSelector = CameraSelector.DEFAULT_FRONT_CAMERA,
     modifier: Modifier = Modifier,
+    cameraSelection: CameraSelection = CameraSelection.DEFAULT_FRONT_CAMERA,
+    captureResolution: CameraCaptureResolution = CameraCaptureResolution.LOW,
+    showCameraPreview: Boolean = true,
```

Comment:
The caller should think about these parameters so please remove the default values here.